docs/index.rst

```diff
@@ -6,15 +6,15 @@ AiiDA ABC tutorial
 +-----------------+-----------------------------------------------------------------------------------------------------------+
 | Related resources                                                                                                           |
 +=================+===========================================================================================================+
-| python packages | `aiida-core 1.4.3`_, `aiida-quantumespresso 3.2.1`_                                                       |
+| python packages | `aiida-core 1.3.0`_, `aiida-quantumespresso 3.0.0`_                                                       |
```

Comment:
Umm, didn't we just set these to the ones from the AiiDAlab cluster? These are the old ones again, right?

Reply:
sorry, don't know what happened.